packages/wrangler/src/config/environment.ts

```diff
@@ -652,14 +660,50 @@ interface EnvironmentNonInheritable {
 			  };
 	};
 
+	/**
+	 * Specifies a list of mTLS certificates that are bound to this Worker environment.
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `[]`
+	 * @nonInheritable
+	 */
 	mtls_certificates: {
-		/** The binding name used to refer to the certificate in the worker */
+		/** The binding name used to refer to the certificate in the Worker */
 		binding: string;
 		/** The uuid of the uploaded mTLS certificate */
 		certificate_id: string;
 	}[];
 
+	/**
+	 * Specifies a list of Tail Workers that are bound to this Worker environment
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `[]`
+	 * @nonInheritable
+	 */
 	tail_consumers?: TailConsumer[];
+
+	/**
+	 * Specifies namespace bindings that are bound to this Worker environment.
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `[]`
+	 * @nonInheritable
+	 */
+	dispatch_namespaces: {
```

Comment:
can we pls confirm this is indeed nonInheritable? The code comment seems to point to that, but the key was listed in the inheritable list

Reply:
In which case it is in the incorrect interface?

Reply:
indeed. will fix that!